build.gradle

```
@@ -71,7 +71,7 @@ tasks.withType(JavaExec).configureEach {
   javaLauncher.set(javaToolchains.launcherFor(java.toolchain))
 }
 
-tasks.withType(Test) {
+tasks.withType(Test).configureEach {
   useJUnitPlatform()
 
   testLogging {
```

It could have been a pre-existing problem, but I find the "test" task doesn't work consistently in my local IntelliJ. Adding these lines for the logging seems to stop IntelliJ getting confused:
```suggestion
  testLogging {
    events "passed", "skipped", "failed"
    exceptionFormat "full"
    showStandardStreams = true
```